.github/workflows/release.yaml

```diff
@@ -1,21 +1,22 @@
 name: Release
 
-# Only trigger this workflow when a tag is pushed in the format `vA.B.C`.
 on:
+  # Trigger this workflow when a tag is pushed in the format `v1.2.3`.
   push:
-    # Syntax: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
     tags:
+      # Pattern syntax: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
       - "v[0-9]+.[0-9]+.[0-9]+*"
+  # Trigger this workflow manually via workflow dispatch.
   workflow_dispatch:
     inputs:
       version:
-        description: 'Version - in the form of v1.2.3'
+        description: 'Version number in the format `v1.2.3`'
         required: true
         type: string
 
 # Configure constants for this workflow.
 env:
-  # TODO: This may become unnecessary: https://github.com/rust-lang/cargo/issues/6790
+  # This may become unnecessary: https://github.com/rust-lang/cargo/issues/6790
```

Comment:
```suggestion
```
I think this comment is more confusing than helpful to a user. Better to setup an issue imo.

Reply:
Okay, fair enough.